django_db_schema/hospital/models.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models
+
+
+class Hospital(models.Model):
+    name = models.CharField(max_length=100, null=False, blank=False)
+    city = models.CharField(max_length=100, null=False, blank=False)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        indexes = [
+            models.Index(fields=['name'])
+        ]
+
+
+class Department(models.Model):
+    hospital = models.ForeignKey(Hospital, on_delete=models.CASCADE,
+                                 related_name="departments", null=False,
+                                 blank=False)
+    name = models.CharField(max_length=100, null=False, blank=False)
+
+    def __str__(self):
+        return self.name
+
+
+class Person(models.Model):
+    name = models.CharField(max_length=100, null=False, blank=False)
+    age = models.PositiveSmallIntegerField(null=True, blank=True)
+    gender = models.CharField(max_length=1,
+                              choices=[('M', 'Male'), ('F', 'Female'),
+                                       ('O', 'Other')], null=False,
+                              blank=False)
+
+    class Meta:
+        abstract = True
+
+    def __str__(self):
+        return self.name
+
+
+class HospitalWorker(Person):
+    POSITION_DOCTOR = 'Doctor'
+    POSITION_NURSE = 'Nurse'
+    departments = models.ManyToManyField(Department, related_name="workers",
```

Comment:
Thats true that `HospitalWorker` can work in multiple departments but in this case they may have different position.
How do you handle this?

Reply:
add the position to the relation table, with "through"

Reply:
Not sure what you meant here, I would like to see an example